packages/manager/modules/billing/src/payment/method/add/routing.js

```diff
@@ -130,13 +130,34 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
               selectedPaymentMethodType?.paymentType ===
                 OVH_PAYMENT_METHOD_TYPE.BANK_ACCOUNT
                 ? 'billing_payment_method_add_sepa_success'
-                : 'billing_payment_method_add_success',
+                : 'billing_payment_method_add_status_success',
             ),
           },
           get($transition$.params(), 'from', null),
         );
       },
-
+      goToPaymentListPage: /* @ngInject */ (
+        $transition$,
+        $translate,
+        goPaymentList,
+      ) => {
+        const { status } = $transition$.params();
+        const MESSAGE_TYPE = {
+          error: 'error',
+          failure: 'error',
+          pending: 'info',
+          cancel: 'error',
+          success: 'success',
+        };
```

Comment:
We can move in an constant file ?

Reply:
This constant is only used in this function. So that's why I did not move it to a constant file.